From 10c245dcb619ce9926b8b063fd1b139029adc686 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 9 Jul 2024 08:44:20 +0200
Subject: [PATCH 1100/1695] drivers: firewall: stm32_etzpc: don't include
 keep.h

Remove useless inclusion of keep.h in stm32_etzpc driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f680275906959613ef962c2afccd204bfa720298 drivers: firewall: stm32_etzpc: don't include keep.h]
Change-Id: I0733ef3fea7dbaf4a06408abbdf1a645d655d485
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/392421
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/drivers/firewall/stm32_etzpc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/drivers/firewall/stm32_etzpc.c b/core/drivers/firewall/stm32_etzpc.c
index 0a51eb258..48c7d836e 100644
--- a/core/drivers/firewall/stm32_etzpc.c
+++ b/core/drivers/firewall/stm32_etzpc.c
@@ -17,7 +17,6 @@
 #include <drivers/stm32mp_dt_bindings.h>
 #include <initcall.h>
 #include <io.h>
-#include <keep.h>
 #include <kernel/boot.h>
 #include <kernel/dt.h>
 #include <kernel/panic.h>
-- 
2.47.2

